src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1249,7 +1249,8 @@ private function registerValidationConfiguration(array $config, ContainerBuilder
 
         $container
             ->getDefinition('validator.not_compromised_password')
-            ->setArgument(2, $config['disable_not_compromised_password'])
+            ->setArgument(2, !$config['not_compromised_password']['enabled'])
```

Comment:
What about renaming the attribute in the validator to `enabled` and swapping the logic there instead of doing the inversion here?

Reply:
Adapted, build failure seems unrelated (TwigBridge errors)